src/briefcase/commands/convert.py

```diff
@@ -0,0 +1,866 @@
+
+        with open(new_pyproject_file) as file:
+            new_pyproject = file.read()
+
+        if pyproject_file.exists():
+            with open(pyproject_file) as file:
+                pyproject = file.read()
+
+            # The pyproject.toml file in the target directory has no briefcase keys, so it's
+            # safe to copy-paste the text, and that way also keep formatting and comments.
+            merged_pyproject = "\n\n\n".join((new_pyproject, pyproject))
+        else:
+            merged_pyproject = new_pyproject
+
+        with open(pyproject_file, "w") as file:
+            file.write(merged_pyproject)
+
+    def migrate_necessary_files(self, project_dir, test_source_dir):
+        """Copy and merge the necessary files from project_dir to target.
+
+        Will warn a LICENSE or CHANGELOG file is missing.
+
+        :param project_dir: The path to the project created by the cookiecutter.
+        :param target: The path where the newly created pyproject.toml file should be migrated to.
+        """
+        self.merge_or_copy_pyproject(project_dir, self.base_path)
+
+        # Copy license file if not already there
+        license_file = self.pep621_data.get("license", {}).get("file")
+        if license_file is not None and Path(license_file).name != "LICENSE":
+            self.logger.warning(
+                f"License file found in {self.base_path}, but its name is not LICENSE. Renaming the file to LICENSE."
+            )
+            (self.base_path / license_file).rename(self.base_path / "LICENSE")
+
+            with open(self.base_path / "pyproject.toml") as f:
+                pyproject_text = f.read()
+            pyproject_text = pyproject_text.replace(license_file, "LICENSE")
+            with open(self.base_path / "pyproject.toml", "w") as f:
+                f.write(pyproject_text)
+
+        elif not (self.base_path / "LICENSE").exists():
+            self.logger.warning(
+                f"License file not found in {self.base_path}. Creating a template LICENSE file "
+            )
+            copy2(project_dir / "LICENSE", self.base_path / "LICENSE")
+
+        # Copy changelog file
+        changelog_file = self.base_path / "CHANGELOG"
+        if not changelog_file.is_file():
+            self.logger.warning(
+                f"Changelog file not found in {self.base_path}. You should either create a new changelog file in"
+                f" {self.base_path} or rename an already existing changelog file to CHANGELOG."
+            )
+
+        # Copy tests or test entry script
+        if test_source_dir:
+            test_entry_script = project_dir / test_source_dir / f"{project_dir.name}.py"
+            copy2(
+                test_entry_script,
+                self.base_path / test_source_dir / f"{project_dir.name}.py",
+            )
+        else:
+            copytree(project_dir / "tests", self.base_path / "tests")
+
+    def setup_existing_app_for_briefcase(
```

Comment:
Following the pattern of other commands, this entry point should be `convert_app()`. The fact that it calls a command called `new_app` that unpacks the app from a template suggests that Convert should extend New.

Reply:
Agreed, we have now changed the name of the entry point to `convert_app`